nafas/params.py

```diff
@@ -19,6 +19,7 @@
 NAFAS_VERSION = "0.6"
 
 SOUND_ERROR_MESSAGE = "ERROR : Unable to play sound."
+SOUND_WARNING_MESSAGE = "Your device is not compatible with our underlaying sound playing library. You can refer to https://github.com/sepandhaghighi/nafas/issues/49 if you wanted to solve it."
```

Comment:
1. Did you mean `underlying`?
2. sound-playing
3. Remove this section: `if you wanted to solve it`

Reply:
Solved in [f0ad4bd2d587](https://github.com/sepandhaghighi/nafas/pull/71/commits/f0ad4bd2d587c4ba75a38c988c9e9b3815754033).